### PR TITLE
Add `Yaz's Summer Cup 2019` tournament article

### PR DESCRIPTION
**Pull request by Bazzadwarf**
Adds all the content to `Yaz's Summer Cup 2019` and fixes a flag in `Yaz's Summer Cup 2018`

- [x] The yaz check™

### CODE REVIEW

**Comment by Bazzadwarf** [comment]
Got the okay from Yaz, ready for a review!

File: wiki/Tournaments/YSC/2019/en.md [hunk 1]
```diff
@@ -16,4 +59,715 @@ The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu!
...
+- Tiebreaker
+  - **[Ito Kanako - Sky of Twilight (Natsu) [Stairway To Heaven]](https://osu.ppy.sh/beatmapsets/677104#osu/1432041)**
+
+### Round of 16
+
+**[Download the mappack here! (165.8 MB)](https://mega.nz/#!v15QmIaa!wCMR-il_t9Zmsob7-zsYHvrEvRGpPi5ROMSfly7rH-I "MEGA")**
+
+- NoMod
+  - [Mimori Suzuko - Light for Knight (\_kotachi\_) [Destiny]](https://osu.ppy.sh/beatmapsets/801337#osu/1682101)
+  - PSYQUI - Still in my heart feat. Punipuni Denki (Nanashima Asami) \[I'm gonna miss you]\
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2019/en.md:216]:
```suggestion
  - PSYQUI - Still in my heart feat. Punipuni Denki (Nanashima Asami) \[I'm gonna miss you\]
```

File: wiki/Tournaments/YSC/2019/en.md [hunk 2]
```diff
@@ -16,4 +59,715 @@ The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu!
...
+| **Yaong** ![][flag_KR] | **6** | 3 | ![][flag_PL] MrBooM | [#1](https://osu.ppy.sh/community/matches/53999915) |
+| Doomsday ![][flag_GB] | 2 | **6** | ![][flag_US] **thaibuy** | [#1](https://osu.ppy.sh/community/matches/54005334) |
+
+#### Losers Bracket Round 4
+
+Friday, 9 August 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Dada ![][flag_BR] | 1 | 6 | ![][flag_JP] Varvalian | [#1](https://osu.ppy.sh/community/matches/53946617) |
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2019/en.md:423]:
```suggestion
| Dada ![][flag_BR] | 1 | **6** | ![][flag_JP] **Varvalian** | [#1](https://osu.ppy.sh/community/matches/53946617) |
```

File: wiki/Tournaments/YSC/2019/en.md [hunk 3]
```diff
@@ -16,4 +59,715 @@ The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu!
...
+Saturday, 10 August 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| YokesPai ![][flag_NO] | 4 | **6** | ![][flag_RO] **badeu** | [#1](https://osu.ppy.sh/community/matches/53970979) |
+| **Bubbleman** ![][flag_GB] | **6** | 3 | ![][flag_AU] Dumii | [#1](https://osu.ppy.sh/community/matches/53973665) |
+| TopSp1n ![][flag_GB] | 0 | **1** | ![][flag_CN] **Start** | *forfeit* |
+| **Nega** ![][flag_GB] | **6** | 2 | ![][flag_CN] Woey | [#1](https://osu.ppy.sh/community/matches/53979256) |
+| lain ![][flag_US] | 2 | **6** | ![][flag_IL] **Xilver15** | [#1](https://osu.ppy.sh/community/matches/53980620) |
+| Dustice ![][flag_DE] | 0 | 1 | ![][flag_PL] GunOfGaming | *forfeit* |
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2019/en.md:434]:
```suggestion
| Dustice ![][flag_DE] | 0 | **1** | ![][flag_PL] **GunOfGaming** | *forfeit* |
```

File: wiki/Tournaments/YSC/2019/en.md [hunk 4]
```diff
@@ -16,4 +59,715 @@ The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu!
...
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Gomo Pslvarh ![][flag_KR] | 1 | **6** | ![][flag_PL] **twoj stary** | [#1](https://osu.ppy.sh/community/matches/53786879) |
+
+Saturday, 3 August 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Varvalian ![][flag_JP] | 1 | 0 | ![][flag_DE] hallowatcher | *forfeit* |
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2019/en.md:559]:
```suggestion
| **Varvalian** ![][flag_JP] | **1** | 0 | ![][flag_DE] hallowatcher | *forfeit* |
```

File: wiki/Tournaments/YSC/2019/en.md [hunk 5]
```diff
@@ -16,4 +59,715 @@ The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu!
...
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Bartek22830** ![][flag_PL] | **5** | 2 | ![][flag_US] Woey | [#1](https://osu.ppy.sh/community/matches/53386508) |
+| Vaxei ![][flag_US] | 0 | **1** | ![][flag_DE] **SgtCataclyst** | *forfeit* |
+| **TopSp1n** ![][flag_GB] | **5** | 4 | ![][flag_US] BeasttrollMC | [#1](https://osu.ppy.sh/community/matches/53388011) |
+
+### Qualifiers
+
+See the [Qualifier Results sheet](https://docs.google.com/spreadsheets/d/1uAaQ949SsbFOXrjw_jczta0rN4ytV4A2ClbTmdX3-RI/edit#gid=27448477 "Google Sheets") for a list of all qualification results.
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2019/en.md:693]:
```suggestion
See the [Qualifiers results sheet](https://docs.google.com/spreadsheets/d/1uAaQ949SsbFOXrjw_jczta0rN4ytV4A2ClbTmdX3-RI/edit#gid=27448477 "Google Sheets") for a list of all qualification results.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**